### PR TITLE
[ENH] Include repetition time in functional summary

### LINKED ISSUE #1425: Add TR to functional summary reports
The TR information appears to be missing from the functional summary reports.

### PR DESCRIPTION
**Pull request by wiheto**
## Changes proposed in this pull request

Closes #1425. Small change that adds the repetition time to the functional reports. 

(Was looking how the reports were generated and added it).

### CODE REVIEW

File: fmriprep/interfaces/reports.py [hunk 1]
```diff
@@ -154,6 +155,7 @@ class FunctionalSummaryInputSpec(BaseInterfaceInputSpec):
                                    mandatory=True)
     output_spaces = traits.List(desc='Target spaces')
     confounds_file = File(exists=True, desc='Confounds file')
+    tr = traits.Float(desc='TR', mandetory=True)
```
Comment (effigies) [inline fmriprep/interfaces/reports.py]:
```suggestion
    tr = traits.Float(desc='Repetition time', mandatory=True)
```

File: fmriprep/interfaces/reports.py [hunk 2]
```diff
@@ -34,6 +34,7 @@
 
 FUNCTIONAL_TEMPLATE = """\t\t<h3 class="elem-title">Summary</h3>
 \t\t<ul class="elem-desc">
+\t\t\t<li>Repetition Time: {tr}</li>
```
Comment (effigies) [inline fmriprep/interfaces/reports.py]:
```suggestion
\t\t\t<li>Repetition time (TR): {tr:.03g}s</li>
```

`{:.03g}` formats floats to up to three decimals, but drops trailing zeros. And we should use units.

**Comment by effigies** [comment]
@wiheto I'm planning to release 1.3.1 today. Do you have time to finish this off?

**Comment by wiheto** [comment]
Can do. I can sit this morning and fix all the stuff you have suggested (here and niworkflows). I’ll get to it in a few hours.

**Comment by oesteban** [comment]
@effigies, why don't we release a 1.3.0-2? after all we needed to fix 1.3.0. That would also give @wiheto some extra time for these fixes (and release 1.3.1 asap).

**Comment by effigies** [comment]
Because we have an actual change in functionality (#1467) already in master.

**Comment by oesteban** [comment]
Yeah, I was going to slack you right now. I had that situation as well before releasing 1.3.0-1. Let me make a backport.

**Comment by effigies** [comment]
We have a patched Docker container people can use for now. I don't know that we need to rush beyond just getting it out today.

**Comment by oesteban** [comment]
It was just cherry-picking your commit based off 1.3.0.post1, it can't hurt. - https://github.com/poldracklab/fmriprep/releases/tag/1.3.0.post2

**Comment by wiheto** [comment]
Committed suggestions. 

Also here is an example of the output of the changes (if making a tmp.tsv confounds file with the confounds a,b,c).

```
from fmriprep.interfaces.reports import FunctionalSummary
summary = FunctionalSummary(slice_timing=True,
                        registration='FreeSurfer',
                        registration_dof=6,
                        pe_direction='i',
                        tr=2,
                        distortion_correction='a string',
                        fallback=False,
                        output_spaces=['MNI152NLin2009cAsym'],
                        confounds_file='tmp.tsv')
summary.run()
```

Output file in ./reports.html is:

```
                <h3 class="elem-title">Summary</h3>
                <ul class="elem-desc">
                        <li>Repetition time (TR): 2s</li>
                        <li>Phase-encoding (PE) direction: Left-Right</li>
                        <li>Slice timing correction: Applied</li>
                        <li>Susceptibility distortion correction: a string</li>
                        <li>Registration: FreeSurfer <code>bbregister</code> (b$
                        <li>Functional series resampled to spaces: MNI152NLin20$
                        <li>Confounds collected: a, b, c</li>
                </ul>
 ```

The only thing I was a slight unsure about is if the flag Mandetory=True should be there. Is there always able to get RepetitionTime in metadata?

**Comment by effigies** [comment]
> The only thing I was a slight unsure about is if the flag Mandetory=True should be there. Is there always able to get RepetitionTime in metadata?

Yes. This is required by BIDS.

**Comment by wiheto** [comment]
Good, then this won't break anything.

**Pull request merged**